src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs

```diff
@@ -395,7 +401,49 @@ private static void ProcessResult(IAsyncResult asyncResult, AsyncCallback asyncC
             HttpInstrumentationEventSource.Log.FailedProcessResult(ex);
         }
 
+        activity.SetEndTime(DateTime.UtcNow);
         activity.Stop();
+
+        TagList tags = default;
+
+        // see the spec https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/trace/semantic_conventions/http.md
+        if (emitOldAttributes)
+        {
+            foreach (var tag in activity.TagObjects)
```

Comment:
this is quite different from how metrics are created for the .NET, .NET Core flavor. In those cases, even if a user does not enable AddHttpClientInstrumentation() on TracerProviderBuilder, metrics would flow fine.

This PR will require AddingHttpClientInstrumentation() done on TracerProviderBuilder, else no metrics would flow. Might be okay, but require an explicit callout in the documentation.

Reply:
100% agree, it'll need a prominent notice that tracing is required. I couldn't see a clean way to just enable metrics without a decent amount of rework and risk. I'd consider this an intermediate step towards fully separate metrics and traces.

FWIW, the auto-instrumentation requires ASP.NET tracing to be enabled to get metrics on .NET Framework, so this would not be a unique situation for those on .NET Framework still.